rskj-core/src/main/java/co/rsk/peg/BridgeMethods.java

```diff
@@ -460,6 +460,18 @@ public enum BridgeMethods {
         fixedPermission(true),
         CallTypeHelper.ALLOW_STATIC_CALL
     ),
+    GET_STATE_FOR_SVP_CLIENT(
+        CallTransaction.Function.fromSignature(
+                "getStateForSvpClient",
+                new String[]{},
+                new String[]{"bytes"}
+        ),
+        fixedCost(4000L),
```

Comment:
Maybe we should review this value later on

Reply:
I don't think it really matters. This function will always be called as `call`. No gas will be spent really. Unless somebody wants to waste gas and uses `send` instead.

Reply:
Anyways, `getStateForBtcReleaseClient` has 4000L and `getStateForDebugging` has `3_000_000L`. Since this is similar to `getStateForBtcReleaseClient`, defining 4000L would be ok.

Reply:
Please leave a TODO in the code and a task in the backlog to review this

Reply:
Created task to review this.